[IMP] demo

tbgracy opened the pull request:
Get app keys from .env file

With the api being actually down that's the only modification I can do now.

rivo2302 commented:
@tbgracy  can you add the example of .env file  in the project and update also the the docs in the README.md.

tbgracy commented:
@rivo2302 i've added an example of a .env file (although it is recommended to always include **.env** file in **.gitignore** for security reasons) and updated the **README.md** as you asked
